ATOI and ITOA/testing_itoa_atoi.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+from atoi_itoa import atoi_itoa
+
+class Test(unittest.TestCase):
+
+    def test_atoi(self):
```

Comment:
Use what we learned about "equivalence class" testing and "invalid cases" from week2 examples to come up with more test cases. For example, what happens in your program if you call atoi("-")?